params.go

```diff
@@ -116,97 +107,82 @@ type ListParams struct {
 	StripeAccount *string `form:"-"` // Passed as header
 }
 
-// ListMeta is the structure that contains the common properties
-// of List iterators. The Count property is only populated if the
-// total_count include option is passed in (see tests for example).
-type ListMeta struct {
-	HasMore    bool   `json:"has_more"`
-	TotalCount uint32 `json:"total_count"`
-	URL        string `json:"url"`
+// AddExpand appends a new field to expand.
+func (p *ListParams) AddExpand(f string) {
+	p.Expand = append(p.Expand, &f)
 }
 
-// RangeQueryParams are a set of generic request parameters that are used on
-// list endpoints to filter their results by some timestamp.
-type RangeQueryParams struct {
-	// GreaterThan specifies that values should be a greater than this
-	// timestamp.
-	GreaterThan int64 `form:"gt"`
-
-	// GreaterThanOrEqual specifies that values should be greater than or equal
-	// to this timestamp.
-	GreaterThanOrEqual int64 `form:"gte"`
-
-	// LesserThan specifies that values should be lesser than this timetamp.
-	LesserThan int64 `form:"lt"`
-
-	// LesserThanOrEqual specifies that values should be lesser than or
-	// equalthis timetamp.
-	LesserThanOrEqual int64 `form:"lte"`
+// GetListParams returns a ListParams struct (itself). It exists because any
+// structs that embed ListParams will inherit it, and thus implement the
+// ListParamsContainer interface.
```

Comment:
Do you consider this type public? 
I guess I have some confusion around what is "public for other packages to use" and what is "public for users to use". And this seems to be the latter right?

Reply:
(I'll cover this in a comment below that responds to your top-level points.)